include/crocoddyl/multibody/actions/contact-fwddyn.hpp

```diff
@@ -119,6 +123,8 @@ struct DifferentialActionDataContactFwdDynamicsTpl : public DifferentialActionDa
   MatrixXs Kinv;
   MatrixXs df_dx;
   MatrixXs df_du;
+  VectorXs x_static;
+  MatrixXs J_static;
```

Comment:
These are temporary variables that are not useful in the API. It would be better not to give them proper names. Rename them to `tmp_XXX` please.

Reply:
Ok, it sounds good to follow this approach. I have modified as suggested, note that there are plenty of places that don't follow it.